cylc/flow/etc/examples/datetime-cycling/flow.cylc

```diff
@@ -0,0 +1,44 @@
+[meta]
+    title = Datetime Cycling
+    description = """
+        A basic cycling workflow which runs the same set of tasks over
+        and over. Each cycle will be given a datetime identifier.
+
+        The task "a" will wait until the real-world (or wallclock) time passes
+        the cycle time.
+
+        Try changing the "initial cycle point" to "previous(00T00) - P1D" to
+        see how this works.
+    """
+
+[scheduling]
+    # start this workflow today at midnight
+    initial cycle point = previous(T00)
```

Comment:
To me this kind of implies the workflow will wait until midnight before starting
```suggestion
    initial cycle point = previous(T00)  # midnight earlier today
```